sdk/storage/Azure.Storage.Files.DataLake/swagger/readme.md

```diff
@@ -0,0 +1,106 @@
+# File Storage
```

Comment:
You'll also need an entry in https://github.com/Azure/azure-sdk-for-net/blob/master/eng/.docsettings.yml#L68 to keep the Analyze step happy

Reply:
Done, but there's more problems.